test/test_helpers.py

```diff
@@ -41,11 +41,14 @@ def cleanup():
 
 
 def fake_script_calls(test_case, clear=False):
-    with open(str(test_case.fake_script_path / 'calls.txt'), 'r+') as f:
-        calls = [line.split(';') for line in f.read().splitlines()]
-        if clear:
-            f.truncate(0)
-        return calls
+    try:
+        with open(str(test_case.fake_script_path / 'calls.txt'), 'r+') as f:
+            calls = [line.split(';') for line in f.read().splitlines()]
+            if clear:
+                f.truncate(0)
+            return calls
+    except FileNotFoundError:
+        return []
```

Comment:
I wanted to add a test that if things fail, they *don't* trigger a call to fake_script, but then you get FileNotFound because calls.txt is created in the actual fake script.
It felt cleaner to assert the calls was an empty list than asserting the file wasn't created.

Reply:
:+1: to this change; I'd done the exact same thing during my dispatch branch but reverted it when it stopped being necessary there (and the change wasn't related to the branch), but it's needed